ocdaction/ocdaction/templates/dashboard/act/task_edit.html

```diff
@@ -0,0 +1,15 @@
+{% extends "layout.html" %}
+
+{% load static %}
+
+{% block main %}
+    <section>
+        <h1>Edit task</h1>
+        <form class="col-xs-6 col-xs-offset-3" method="POST" action="{{ task_form.get_absolute_url }}">
+            {% csrf_token %}
```

Comment:
Please could you convert this form to use widget tweaks? Example here it is mostly copy paste :D https://github.com/womenhackfornonprofits/ocdaction/blob/904134b35599e0d7f03b97db7b9280236d2c803e/ocdaction/ocdaction/templates/profiles/login.html